images_to_samples.py

```diff
@@ -326,7 +329,8 @@ def main(params):
 
     final_samples_folder = None
 
-    sample_path_name = f'samples{samples_size}_overlap{overlap}_min-annot{min_annot_perc}_{num_bands}bands'
+    run_name = get_key_def('mlflow_run_name', params['global'], default='gdl')
+    sample_path_name = f'samples{samples_size}_overlap{overlap}_min-annot{min_annot_perc}_{num_bands}bands_{run_name}'
```

Comment:
In our case, an mlflow run is a single training, right? Since multiple runs (ex.: "test13_unet", "test13_deeplabv3", etc.) can share the same samples, should the samples really be named after the run? It seems to me the experiment name might be more relevant, no?

Reply:
I am also struggling with this, I started using the same run_name when samples are shared. I agree this is am ambiguous use of the concept of mlflow run. I wouldn't mind changing this to the experiment name...